src/frontend/apps/standalone_site/src/features/Contents/features/Video/components/Update/VideoUpdate.tsx

```diff
@@ -0,0 +1,73 @@
+import { AppConfig, AppConfigProvider, UploadManager } from 'lib-components';
+import { useVideo, DashboardVideoWrapper, useSetVideoState } from 'lib-video';
+import 'lib-video/lib/esm/style.css';
+import { defineMessages, useIntl } from 'react-intl';
+import { useParams } from 'react-router-dom';
+
+import liveBackground from 'assets/img/liveBackground.jpg';
+import { REACT_QUERY_CONF_API } from 'conf/global';
+import { ManageAPIState } from 'features/Contents';
+
+const appConfig = {
+  static: {
+    img: {
+      liveBackground,
+    },
+  },
+} as AppConfig;
+
+const messages = defineMessages({
+  NoVideo: {
+    defaultMessage: 'There is no video to display.',
+    description: 'Text when there is no video to display.',
+    id: 'features.Contents.features.Video.Dashboard.NoVideo',
+  },
+});
+
+const VideoUpdate = () => {
+  const intl = useIntl();
+  const { videoId } = useParams<{ videoId?: string }>();
+
+  if (!videoId) {
+    return (
+      <ManageAPIState
+        isError={false}
+        isLoading={false}
+        hasResult={false}
+        nothingToDisplay={intl.formatMessage(messages.NoVideo)}
+      />
+    );
+  }
+
+  return <VideoDashboard videoId={videoId} />;
+};
+
+const VideoDashboard = ({ videoId }: { videoId: string }) => {
+  const intl = useIntl();
+  const {
+    isError,
```

Comment:
Is it possible to fine tune the error ? If I tried to fetch a non existing video, the error message is not relevant.

Reply:
https://github.com/openfun/marsha/pull/2086/commits/556c344ca6ff148864c6480d84388587cf7b4710#diff-d5e7c8a2047ffd1d224e1b7a64dbc145099104138bd16469da576950b5fd1ee1R70